tests/tests_pytorch/utilities/test_data.py

```diff
@@ -289,3 +290,23 @@ def __init__(self, indices=None, **kwargs):
     dataloader = ArrayAttributeDataloader(dataset)
     dl_args, dl_kwargs = _get_dataloader_init_args_and_kwargs(dataloader, dataloader.sampler)
     assert dl_kwargs["indices"] is dataloader.indices
+
+
+def test_set_sampler_epoch():
```

Comment:
Wouldn't the test go to test_fabric/utilities/test_data?

Reply:
Oops! Good catch. That's what I intended